src/modules/staking/models/staking.args.ts

```diff
@@ -39,6 +39,8 @@ export class ClaimRewardsWithNewValueArgs extends GenericStakeFarmArgs {
 export class StakingFarmsFilter {
     @Field(() => String, { nullable: true })
     searchToken?: string;
+    @Field({ nullable: true })
+    rewardsDepletedPerFarm?: boolean;
```

Comment:
maybe rename into `rewardsEnded`. `depleted` would means that all rewards were distributed and it wouldn't take into account the manual stop of rewards

Reply:
done